frontend/src/fetchers/recipientSpotlight.js

```diff
@@ -4,8 +4,8 @@ import { get } from './index';
 
 const recipientUrl = join('/', 'api', 'recipient-spotlight');
 
-export const getRecipientSpotlight = async (sortBy = 'recipientName', sortDir = 'desc', offset = 0, filters, limit = 10) => {
-  const limitParam = limit !== undefined && limit !== null ? `&limit=${limit}` : '';
+export const getRecipientSpotlight = async (sortBy = 'recipientName', sortDir = 'desc', offset = 0, filters, limit = undefined) => {
```

Comment:
```
export const getRecipientSpotlight = async (sortBy = 'recipientName', sortDir = 'desc', offset = 0, filters, limit)
```

no need for explicit undefined, I would think

Reply:
i guess does it change anything?